lib/Command/PageTrashCleanup.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OCA\Collectives\Command;
+
+use OC\Core\Command\Base;
+use OCA\Collectives\Db\CollectiveMapper;
+use OCA\Collectives\Service\MissingDependencyException;
+use OCA\Collectives\Service\NotFoundException;
+use OCA\Collectives\Service\NotPermittedException;
+use OCA\Collectives\Trash\PageTrashBackend;
+use OCP\App\IAppManager;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+class PageTrashCleanup extends Base {
+	private ?PageTrashBackend $trashBackend = null;
+	private CollectiveMapper $collectiveMapper;
+
+	public function __construct(
+		IAppManager $appManager,
+		CollectiveMapper $collectiveMapper
+	) {
+		parent::__construct();
+		if ($appManager->isEnabledForUser('files_trashbin')) {
+			$this->trashBackend = \OC::$server->get(PageTrashBackend::class);
+		}
+		$this->collectiveMapper = $collectiveMapper;
+	}
+
+	protected function configure(): void {
+		$this
+			->setName('collectives:pages:trashbin:cleanup')
+			->setDescription('Empty the collectives page trashbin.')
+			->addArgument('collective', InputArgument::OPTIONAL, 'name of collective')
+			->addOption('force', 'f', InputOption::VALUE_NONE, 'skip confirmation');
+		parent::configure();
+	}
+
+	/**
+	 * @param InputInterface  $input
+	 * @param OutputInterface $output
+	 *
+	 * @return int
+	 */
+	protected function execute(InputInterface $input, OutputInterface $output): int {
+		if (!$this->trashBackend) {
+			$output->writeln('<error>files_trashbin is disabled: collectives page trashbin is not available</error>');
+			return 1;
+		}
+		$helper = $this->getHelper('question');
+
+		$collectives = $this->collectiveMapper->getAll();
```

Comment:
The problem is that we don't have direct access to the collective name in the database. The name only derives from the name of the associated circle. In most places we can limit this to iterate over the collectives known to a user, but since this is a command run by admins, I think we really have to iterate over all collectives in this case.